lib/theme.dart

```diff
@@ -1,50 +1,9 @@
-import 'dart:convert';
 
 import 'package:flutter/material.dart';
 
-final lightTheme = ThemeData(
-    // scaffoldBackgroundColor: Colors.black,
-    // primaryColor: Colors.white,
-    // backgroundColor: Color(0xff181a1b),
-    // colorScheme: const ColorScheme(
-    //     primary: Color(0xffcdcbc9),
-    //     secondary: Colors.grey,
-    //     onPrimary: Colors.blueGrey,
-    //     onSecondary: Color(0xff303035),
-    //     background: Color(0xff303035),
-    //     surface: Colors.black54,
-    //     onSurface: Color(0xffcdcbc9),
-    //     onBackground: Color(0xffcdcbc9),
-    //     error: Colors.red,
-    //     brightness: Brightness.dark,
-    //     onError: Color(0xffcdcbc9))
-    );
-
-final darkTheme = ThemeData(
-    scaffoldBackgroundColor: Colors.black,
-    primaryColor: Colors.white,
-    backgroundColor: Color(0xff181a1b),
-    colorScheme: const ColorScheme(
-        primary: Color(0xffcdcbc9),
-        secondary: Colors.grey,
-        onPrimary: Colors.blueGrey,
-        onSecondary: Color(0xff303035),
-        background: Color(0xff303035),
-        surface: Colors.black54,
-        onSurface: Color(0xffcdcbc9),
-        onBackground: Color(0xffcdcbc9),
-        error: Colors.red,
-        brightness: Brightness.dark,
-        onError: Color(0xffcdcbc9)),
-    sliderTheme: ThemeData.dark().sliderTheme.copyWith(
-        valueIndicatorColor: Colors.black,
-        valueIndicatorTextStyle:
-            TextStyle(color: Colors.grey, backgroundColor: Colors.transparent)),
-    textTheme: TextTheme(
-      labelSmall: TextStyle(fontSize: 12, color: Colors.red),
-    ));
-
+final lightTheme = ThemeData.light();
```

Comment:
how do we customize the theme if we are using .light() and .dart() methods.

Reply:
`ThemeData.dark().copyWith(accentColor: Colors.black)`
I think this will directly override default dark mode
https://stackoverflow.com/questions/65501437/changing-one-property-of-themedata-dark

Reply:
Please override properties based on your solution with .copyWith.